src/main/java/com/mraof/minestuck/blockentity/machine/PunchDesignixBlockEntity.java

```diff
@@ -104,32 +134,56 @@ else if (!player.getInventory().add(getCard()))
 		}
 	}
 	
-	private void handleKeyboardClick(ServerPlayer player)
+	private void handleKeyboardClick(Player player)
 	{
 		ItemStack heldStack = player.getMainHandItem();
-		if(heldStack.getItem() != MSItems.CAPTCHA_CARD.get())
-			return;    //Not a valid item in hand
 		
-		if(getCard().getItem() == MSItems.CAPTCHA_CARD.get())
+		if(player.level.isClientSide)
 		{
-			if(AlchemyHelper.hasDecodedItem(heldStack))
+			//Go to screen if no valid item in hand or crouching
+			//TODO if the player is holding a captcha card and is crouching, it wont open the screen
+			if(player.isShiftKeyDown() || heldStack.getItem() != MSItems.CAPTCHA_CARD.get())
```

Comment:
Could use help with this issue

Reply:
What might be happening is that the item gets priority by shifting, and then consumes the action by returning a non-pass result from the `use()` function. Does that seem correct?

One way to deal with it is to override `item.doesSneakBypassUse()`. Think about if that'd be approriate.

Reply:
Sounds logical! If thats the case, it does not sound appropriate. Extra crouch functionality is low priority in my book

Reply:
Removed